framework/test/gtest/BundleManifestTest.cpp

```diff
@@ -191,3 +217,216 @@ TEST(BundleManifestTest, ParseManifest)
   framework.Stop();
   framework.WaitForStop(std::chrono::milliseconds::zero());
 }
+
+namespace cppmicroservices {
+
+struct TestBundleAService
+{
+  virtual ~TestBundleAService() {}
+};
+
+}
+
+TEST(BundleManifestTest, DirectManifestInstall)
```

Comment:
Will add.

Reply:
I'm not sure this makes sense. If I Install with a non-existent location, InstallBundles throws with the install operation failing... starting at that point would fail because the bundle was never installed.

Reply:
BUT... thanks for addressing the need for negative test here. It turned out that there was a difference in behavior when a manifest is passed in vs. when it's read from the file at location. If a manifest was passed in, the Exists(location) test was never done, but it should have been. I have a test cases the ensures this now. Thanks again!